x/plans/types/plan.go

```diff
@@ -8,6 +8,10 @@ import (
 
 // Function to validate a plan object fields
 func (p Plan) ValidatePlan() error {
+	// validate denom is not empty
+	if p.GetPrice().Denom == "" {
```

Comment:
let's check it's ulava instead of not empty

Reply:
done. see https://github.com/lavanet/lava/commit/a5a242c14ba7ab965e66f4cee60f405ac5ef942d